README.md

```diff
@@ -23,6 +23,7 @@ Please feel free to contribute to this resource list.  To do so, either:
 * [Linkedin](https://www.linkedin.com/jobs/): Lots of job postings here as well as a great place to start cold calling recruiters
 * **Career Fairs**: Don't just hand your resume out, **make sure to get recruiters' contact info** so that you can follow up with them after the career fair as well as next year
 * [Indeed](https://www.indeed.com/): For Job Postings!
+*[Intershala](https://internshala.com/): Internships positions 
```

Comment:
Thanks for the suggestions! However, this seems to be a platform that is focused on internships in India.  This webpage is designed to be more global in nature and as such, this does not seem like an appropriate resource to be added to this README.  Feel free to create your own webpage/README with city specific information if desired.